druid-shell/src/lib.rs

```diff
@@ -47,12 +45,13 @@ pub use clipboard::{Clipboard, ClipboardFormat, FormatId};
 pub use common_util::Counter;
 pub use dialog::{FileDialogOptions, FileInfo, FileSpec};
 pub use error::Error;
-pub use hotkey::{HotKey, KeyCompare, RawMods, SysMods};
-pub use keyboard::{KeyEvent, KeyModifiers};
-pub use keycodes::KeyCode;
+pub use hotkey::{key_event_for_test, HotKey, IntoKey, RawMods, SysMods};
 pub use menu::Menu;
 pub use mouse::{Cursor, MouseButton, MouseButtons, MouseEvent};
 pub use scale::{Scalable, Scale, ScaledArea};
 pub use window::{
     IdleHandle, IdleToken, Text, TimerToken, WinHandler, WindowBuilder, WindowHandle,
 };
+
+// Discussion question: export types to top-level? Just KeyboardEvent?
+pub use keyboard_types;
```

Comment:
I agree with you that re-exporting would be following current practices. However, `Key` in particular collides with the item of the same name in `env`. So I believe doing this would force the submodule discussion.

Reply:
I might just reexport `Key` as `KeyCode` for now?

Reply:
It's possible but I have a pretty strong aversion to having two names for the same type.